packages/engine/Source/Scene/ITwinData.js

```diff
@@ -71,4 +72,67 @@ ITwinData.createTilesetFromIModelId = async function (iModelId, options) {
   return Cesium3DTileset.fromUrl(resource, options);
 };
 
+/**
+ * Create a tileset for the specified reality data id. This function only works
+ * with 3D Tiles meshes and point clouds.
+ *
+ * If the <code>type</code> or <code>rootDocument</code> are not provided this function
+ * will first request the full metadata for the specified reality data to fill these values.
+ *
+ * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
+ *
+ * @param {string} iTwinId The id of the iTwin to load data from
+ * @param {string} realityDataId The id of the reality data to load
+ * @param {ITwinPlatform.RealityDataType} [type] The type of this reality data
+ * @param {string} [rootDocument] The path of the root document for this reality data
+ * @returns {Promise<Cesium3DTileset>}
+ */
+ITwinData.createTilesetForRealityDataId = async function (
+  iTwinId,
+  realityDataId,
+  type,
+  rootDocument,
```

Comment:
I believe type and rootDocument should be extracted from the reality data metadata to ensure that they are consistent with what actually exist in the storage container... or maybe this is our way to avoid a call to get metadata since they were already extracted and assumed consistent?

Reply:
> or maybe this is our way to avoid a call to get metadata since they were already extracted and assumed consistent?

@matmarchand You are correct. If either of these values is missing we will fetch the metadata needed to load the tileset. This will likely only the rootDocument commonly but we grab and update both from the new request anyway.

However if the application using this function already has these values we can skip that network call. This could happen if the app [loads the list of reality data](https://developer.bentley.com/apis/reality-management/operations/get-all-reality-data/) using `return=representation` instead of `return=minimal`. Or if the app already called the individual metadata route themselves. No need to waste extra time and bandwidth if the info is already accessible